services/backend/src/service/project_strategy.js

```diff
@@ -42,7 +44,21 @@ module.exports = (strategyPersistence) => ({
       error.code = 400;
       throw error;
     }
-
-    return strategyPersistence.findSelectedStrategiesGeoJson(pId);
+    const selectedStrategies = await strategyPersistence.findSelectedStrategiesGeoJson(pId);
```

Comment:
this line should be inside a `try - catch`

Reply:
Done